powersimdata/data_access/context.py

```diff
@@ -23,3 +24,16 @@ def get_data_access(data_loc=None):
         if mode == DeploymentMode.Server:
             return SSHDataAccess(root)
         return LocalDataAccess(root)
+
+    @staticmethod
+    def get_launcher(scenario):
+        """Return instance for interaction with simulation engine
+
+        :param powersimdata.Scenario scenario: a scenario object
```

Comment:
Do you know if Sphinx will be able to link to `powersimdata.scenario.scenario.Scenario`?

Reply:
Also, `:return:` is missing

Reply:
You're right, fixed. Also, the return needed a new syntax in order to generate a link to the class